recipe/meta.yaml

```
@@ -9,16 +9,19 @@ package:
 source:
   - url: https://github.com/cupy/cupy/archive/v{{ version }}.tar.gz
     sha256: {{ sha256 }}
+    patches:
+      # Include full CUDA 10.2 support: https://github.com/cupy/cupy/pull/3218
+      - fixes_3218.diff
 
-  # CUB support has too many bugs, let us wait for a bit longer   
+  # CUB support has too many bugs, let us wait for a bit longer
   ## 1. The CUB support has been extensively tested with v1.8.0, so we should pin it here
   ## 2. This dependency should be removed once CuPy bundles CUB (cupy/cupy#2584)
   #- url: https://github.com/NVlabs/cub/archive/v1.8.0.tar.gz
   #  sha256: 025658f4c933cd2aa8cc88a559d013338d68de3fa639cc1f2b12cf61dc759667
   #  folder: cub_v1.8.0  # so the source code is in cupy/cub_v1.8.0
 
 build:
-  number: 1
+  number: 2
```

I am going to modify this single line shortly, as we just bumped to 2 in #42 and I am not sure why this doesn't cause conflict. 
```suggestion
  number: 3
```

---

OK after committing this the conflict appeared...Resolved in 87a95d5. (Sorry @rlratzel for doing the rebase on your behalf 😅)